src/Chunk.h

```diff
@@ -375,12 +375,12 @@ class cChunk :
 		m_IsSaving = false;
 	}
 
-	/** Sets the blockticking to start at the specified block. Only one blocktick may be set, second call overwrites the first call */
-	inline void SetNextBlockTick(int a_RelX, int a_RelY, int a_RelZ)
+	/** Causes the specified block to be ticked on the next Tick() call.
+	Plugins can use this via the cWorld:SetNextBlockToTick() API.
+	Only one block coord per chunk may be set, a second call overwrites the first call */
+	inline void SetNextBlockToTick(const Vector3i a_RelPos)
```

Comment:
What use is an API that only allows ticking one block per chunk anyway?

Reply:
Good enough for a `magic sTick`, in Debuggers :)